### ISSUE TITLE
Incremental patcher

### ISSUE THREAD

**Issue by sparkle-preference** [sparkle-preference/OriWotwRandomizerClient#63]
we really shouldn't make users download the entire wotwrando.exe every patch.

**Comment by MunWolf** [comment]
We now have a new installer and when speaking with Zre he had this to say about it

1. We would either need incremental installers for every previous version (because users may have skipped some versions) OR require the users to download every single update one by one OR create something custom
2. I don't see a problem with downloading 70M, we could however delete the installer after installing through the auto-updater
3. It just creates a ton of possible edge cases and problems that can happen and that we have to have in mind and handle properly (mainly the ones from 1.).

So in short, probably wont happen.
Closing
